src/components/pages/blog-post/blog-post.tsx

```diff
@@ -12,12 +16,31 @@ import { BlogHero } from '../../content/heroes/blog-hero';
 import { LeadboxProps } from '../../content/leadbox/leadbox';
 import { LONG_DATE_FORMAT, useLocaleFormat } from '../../i18n-helpers';
 import { Layout } from '../../layout/layout';
+import { scrollToTarget } from '../../layout/use-anchor-tag-scrolling';
 import customComponents from '../../legacy/markdown/custom-components';
 import { MarkdownAst } from '../../legacy/markdown/markdown-ast';
 import { up } from '../../support/breakpoint';
 import FollowPanel from './follow-panel';
 import SharePanel from './share-panel';
 
+interface ContentfulFootnoteReference
+  extends ContentfulRichTextGatsbyReference {
+  note: any; // TODO: Improve typing here
+}
+
+enum ContentfulCustomModel {
```

Comment:
Good idea to collect those identifiers here.
I would love to have embed asset enums for image, video etc, just for the sake of aligning our approaches. Or do those constants have some enum in the contentful package itself

Reply:
We are splitting the files `contentType` (mime type), Contentful does not provide any enums for this, lets create our own.

Reply:
Saw that, thanks!